build/do.rq

```diff
@@ -215,6 +215,18 @@ run(cmd) {
 	print(out.stdout)
 }
 
+run_quiet(cmd) {
```

Comment:
I guess one option is to just make this `run`?

Reply:
So the difference is that this will print the raw stdout and that allows GH to comment on lines again even when there's an error?